### DISCUSSION TITLE
Ability to set sourceRoot of tailwind.config.js

### DISCUSSION THREAD

**Discussion by k3nsei** [ben-rogerson/twin.macro#307] [Ideas]
I have a monorepo and currently I need to put `tailwind.config.js` in root of my monorepo.
But I want to have ability to put `tailwind.config.js` for every project in it's own location.

**Comment by ben-rogerson** [comment]
I think this is possible right now, you'll just need a babel macros config that uses a path.resolve for the config otherwise it'll use the run location for your scripts as the base location.

```js
// babel-plugin-macros.config.js
const { resolve } = require("path");

module.exports = {
  twin: {
    preset: "styled-components",
    config: resolve(__dirname, "tailwind.config.js"),
  },
};
```

**Comment by k3nsei** [comment]
If so then it should be in documentation.

**Reply by ben-rogerson** [reply]
Not wrong there!
